templates/partials/summary/summary.html

```diff
@@ -42,7 +42,7 @@
     {% do summary_sections.append
       (
         {
-          "summaryTitle": section.title if summary_type == "Summary" and summary_groups else None,
+          "summaryTitle": section.title if summary_type == "Summary",
```

Comment:
Can you clarify why we do this?. Why is it set for `Summary` and not others?

Reply:
Without this on a section summary the section title gets repeated e.g.
<img width="820" alt="Screenshot 2023-05-12 at 12 49 48" src="https://github.com/ONSdigital/eq-questionnaire-runner/assets/47635349/82e58cf9-43dc-4479-a335-834344d70adf">

Reply:
Ah I see, because we set the page title differently for section summaries?

Reply:
Yeah, i've accidentally pushed it back up with it removed, should I add it back?

Reply:
Is it the first or the section one that is this one?. What happens if you remove the separate title on section summary?

Reply:
Could do that but then on the section summary I think the title would be a h2 rather than h1

Reply:
Okay, revert it and keep it as h1

Reply:
Reverted now 👍